filetracker/interaction_test.py

```diff
@@ -104,13 +102,13 @@ def test_file_version_should_return_modification_time(self):
 
         self.client.put_file('/version.txt', src_file)
 
-        remote_path = os.path.join(
-                self.server_dir, 'files', 'files', 'version.txt')
-        modification_time = int(os.stat(remote_path).st_mtime)
+        modification_time = int(os.stat(src_file).st_mtime)
```

Comment:
I'll wait for a comment from @accek on what is the intended behaviour.
But this test should definitely be modified in some way.

Reply:
If we add an old file to filetracker, the addition operation would succeed, but the old file would be silently discarded.

Reply:
So the modification time should be based on the file and not current time, right?
Because it seems the current implementation did not adhere to that assumption, so I guess I should change that and add this delay in the tests to see if it's correct.
(I think the delay could be replaced with utime setting an earlier date in the yedt)

Reply:
Well, I don't get something. When adding a file, the current time should be used as revision (as we may be even streaming something which is not necessarily a file). If there is a file with future modification date already there, we should not replace it. Is it this case you're talking about?

Reply:
Yeah, something like this.